Catappult/appcoins-wallet-android
# [feature/APPC-2793] Wallet Transactions Images

**What does this PR do?**

   Adds new images to wallet history transfers and changes labels.

**How should this be manually tested?**

  Check if the new transfer icons are okay and labels are also as expected (both title and subtitle), you also need to test migrations to see how old transfer are rendered.

**What are the relevant tickets?**

  [APPC-2793](https://aptoide.atlassian.net/browse/APPC-2793)

**Code Review Checklist**
